core/src/test/scala/com/salesforce/op/dsl/RichNumericFeatureTest.scala

```diff
@@ -134,15 +134,55 @@ class RichNumericFeatureTest extends FlatSpec with FeatureTestBase with RichNume
     // TODO: add vectorize() test
   }
 
-  Spec[RichRealNNFeature] should "have tests" in {
-    // TODO: add tests
+  Spec[RichRealNNFeature] should "perform math functions correctly" in {
```

Comment:
thanks for the tests!

Reply:
Is there an easy way to use our `OpTransformerSpec` here, or would we have to make a separate test class for each transformer we're trying to test?